src/login/tests/LoginPage.test.jsx

```diff
@@ -55,6 +55,9 @@ describe('LoginPage', () => {
         secondaryProviders: [],
       },
     },
+    register: {
+      validationApiRateLimited: false,
+    },
```

Comment:
why do we need it in LoginPage test?

Reply:
We have modified and put validations in the Password field, although this would be used only when the password field is getting rendered from register page but as this is in password field, every component rendering password field would require it in its tests.